cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -513,7 +515,7 @@ makeSignedTransaction witnesses (ShelleyTxBody era txbody txmetadata) =
           txbody'
           (Shelley.WitnessSet
             (Set.fromList [ w | ShelleyKeyWitness _ w <- witnesses ])
-            (Map.fromList [ (Shelley.hashScript sw, sw)
+            (Map.fromList [ (Shelley.hashScript @ledgerera sw, sw)
```

Comment:
`Shelley.hashScript` is ambiguous. I'm not a big fan of ambiguous types, but the ledger uses them.